README.md

```diff
@@ -1 +1,76 @@
+**European Variation Archive (EVA) Web Front-end**
```

Comment:
Not bold text, but title header (prefix with # ). Applies to other headers, see README raw file in https://raw.githubusercontent.com/EBIvariation/eva-pipeline/develop/README.md for reference.